susemanager-utils/susemanager-sls/src/grains/virt.py

```diff
@@ -0,0 +1,14 @@
+import salt.modules.virt
+
+
+def __virtual__():
+    return salt.modules.virt.__virtual__()
+
+
+def features():
+    """returns the features map of the virt module"""
+    return {
+        "virt": {
```

Comment:
I wonder if `virt` name for this custom grain is too generic and might collide at some point with some other Salt grain.

What do you think about something like:
```suggestion
        "virt_capabilities": {
```

Reply:
I have been pondering that... then I'll change to `virt_features` rather than `virt_capabilities` since capabilities has another meaning in the virt language.